### PR TITLE
script help formatting

### PR DESCRIPTION
**Pull request by mr-c**
From a fairly thorough review of the help output of all the scripts.

Has the fix for #1257 pulled from #1255

### CODE REVIEW

**Comment by mr-c** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)
- [ ] Is the Copyright year up to date?

**Comment by mr-c** [comment]
@ctb @camillescott ready for review and merge

**Comment by mr-c** [comment]
Rebased on master and again ready for review & merge @camillescott ; thanks!

**Comment by mr-c** [comment]
Jenkins, retest this please

**Comment by mr-c** [comment]
Again made ready for review & merge @camillescott @ctb 

Note: I've disabled the 'oxli' script for the v2.0 release. I will re-enable it right after the release is finished.

**Comment by ctb** [comment]
On Mon, Aug 24, 2015 at 10:17:53PM -0700, Michael R. Crusoe wrote:
> Note: I've disabled the 'oxli' script for the v2.0 release. I will re-enable it right after the release is finished.

Why?

**Comment by mr-c** [comment]
@ctb It isn't ready for non-devs and it currently gets installed very visibly. No need to confuse anyone. One simple PR will return it for further development after 2.0 is released.

File: scripts/do-partition.py [hunk 1]
```diff
@@ -41,10 +41,10 @@
 DEFAULT_K = 32
 
 
-def worker(queue, basename, stop_big_traversals):
+def worker(que, basename, stop_big_traversals):
```
Comment (bocajnotnef) [inline scripts/do-partition.py]: Why this change?

File: setup.py [hunk 2]
```diff
@@ -179,18 +179,18 @@ def check_for_openmp():
...
         "extras_require": {':python_version=="2.6"': ['argparse>=1.2.1'],
                            'docs': ['sphinx', 'sphinxcontrib-autoprogram'],
                            'tests': ['nose >= 1.0']},
         "scripts": SCRIPTS,
-        "entry_points": {
-            'console_scripts': [
-                "oxli = oxli:main"
-            ]
-        },
+        # "entry_points": { # Not ready for distribution yet.
```
Comment (bocajnotnef) [inline setup.py]: Is this (and the disabled tests) just for 2.0?

File: scripts/do-partition.py (hunk 1 above)
Comment (mr-c) [inline scripts/do-partition.py]: Fixes https://github.com/dib-lab/khmer/issues/1257
`queue` was hiding the module `queue`

File: doc/run-corn-50m.sh [hunk 3]
```diff
@@ -12,8 +12,10 @@
 #      https://s3.amazonaws.com/public.ged.msu.edu/khmer/iowa-corn-50m.fa.gz
 #
 
+set -e
+set -x
```
Comment (camillescott) [inline doc/run-corn-50m.sh]: What are these?

Comment (mr-c) [inline doc/run-corn-50m.sh]: Prints out each line as it is executing and stops on first error

File: scripts/partition-graph.py [hunk 4]
```diff
@@ -36,10 +39,10 @@
 DEFAULT_N_THREADS = 4
 
 
-def worker(queue, basename, stop_big_traversals):
+def worker(que, basename, stop_big_traversals):
```
Comment (camillescott) [inline scripts/partition-graph.py]: How about renaming this to something descriptive instead of this hamfisted truncation?

Comment (mr-c) [inline scripts/partition-graph.py]: ... Any suggestions?

File: setup.py (hunk 2 above)
Comment (mr-c) [inline setup.py]: Correct

File: scripts/partition-graph.py (hunk 4 above)
Comment (camillescott) [inline scripts/partition-graph.py]: worker_queue, partition_queue

**Comment by camillescott** [comment]
cool, LGTM

**Comment by mr-c** [comment]
Thanks! Merge?

**Comment by ctb** [comment]
On Tue, Aug 25, 2015 at 04:06:26PM -0700, Michael R. Crusoe wrote:
> @ctb It isn't ready for non-devs and it currently gets installed very visibly. No need to confuse anyone. One simple PR will return it for further development after 2.0 is released.

OK.  Please let's expose these decisions with explanations in either
an issue or in the pull request...
